web-app/src/games/api.admin.games.pipes.ts

```diff
@@ -0,0 +1,12 @@
+import { PipeTransform, Injectable, ArgumentMetadata } from '@nestjs/common';
+
+@Injectable()
+export class JsonPipe<T> implements PipeTransform<any, T> {
```

Comment:
Do you agree to make it more explicit that this pipe is for data transformation only and not validation ?

Reply:
Renamed it to `SimpleJsonParsePipe`, do you think it is enough to make this clear?